reana_client/utils.py

```diff
@@ -173,6 +180,49 @@ def _prepare_kwargs(reana_yaml):
         raise e
 
 
+def _validate_environment_tags(reana_yaml):
+    """Validate environments in REANA specification file according to workflow type.
+
+    :param reana_yaml: Dictionary which represents REANA specifications file.
+    :return: True if environment tags are valid.
+    """
+    workflow_type = reana_yaml["workflow"]["type"]
+
+    if workflow_type == "serial":
+        workflow_steps = reana_yaml["workflow"]["specification"]["steps"]
+        _validate_serial_workflow_environment(workflow_steps)
+    elif workflow_type == "yadage":
+        logging.info("Not Implemented yet.")
```

Comment:
The logging info would not be visible to the user unless they raise the loglevel... which is not probable.

You can simply use `warning`, as it it were a finding, just to emit the message saying "The environment validation is not implemented for Yadage workflows yet."

This would be better, otherwise people may be mislead into a false security that the images were checked and not found problematic.